packages/streams/gpsd.js

```diff
@@ -40,16 +40,18 @@ function Gpsd(options) {
 
   const port = options.port || 2947
   const hostname = options.hostname || options.host || 'localhost'
+  const noDataReceivedTimeout = options.noDataReceivedTimeout || 0
```

Comment:
Is zero really a good default value? Won't it trigger needless reconnects?

Reply:
Hmm.. I wrote a comment here earlier - so setting either reconnectThreshold or reconnectInterval to zero effectively disables the reconnect mechanism. So the default now is to not have it - which is consistent with the UI having no value in the field.